package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "scripture-burrito",
+    "version": "0.2.0",
```

Comment:
I think this should reflect what our next release will be, which should be `0.2.0-beta.2`.